Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -256,24 +256,26 @@ void ABenchmarkGymGameModeBase::Tick(float DeltaSeconds)
 		PrintMetricsTimer.SetTimer(10);
 	}
 #if	STATS
-	if (StatStartFileTimer.HasTimerGoneOff())
+	if (CPUProfileInterval > 0 && StatStartFileTimer.HasTimerGoneOff())
 	{
 		FString Cmd(TEXT("stat startfile"));
 		if (GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking())
 		{
 			USpatialNetDriver* SpatialDriver = Cast<USpatialNetDriver>(GetNetDriver());
-			if (ensure(SpatialDriver != nullptr))
+			if (SpatialDriver != nullptr)
```

Comment:
why was there an ensure here previously?

Reply:
I guess it's from Jungting's previous code...